src/main/java/io/apicurio/datamodels/core/validation/ValidationRule.java

```diff
@@ -252,6 +252,7 @@ protected boolean isValidUrl(String propertyValue) {
      * @param propertyValue
      */
     protected boolean isValidUrlTemplate(String propertyValue) {
+        // isn't this the same as AaiInvalidChannelPathRule#RFC_6570_URI_TEMPLATE
```

Comment:
I think the answer to this is a solid "maybe".  :)  This method is called by the validator for Server URL.  Here is the link to the relevant part of the OpenAPI spec:  

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#serverObject

This is very similar to RFC 6570 but the OpenAPI spec doesn't specify.  Which is weird because RFC 6570 is referenced elsewhere in the OpenAPI spec in an unrelated section.